clients/src/test/java/org/apache/kafka/common/compress/KafkaLZ4Test.java

```diff
@@ -113,9 +116,11 @@ public Stream<? extends Arguments> provideArguments(ExtensionContext context) {
             for (Payload payload : payloads)
                 for (boolean broken : Arrays.asList(false, true))
                     for (boolean ignore : Arrays.asList(false, true))
-                        for (boolean blockChecksum : Arrays.asList(false, true))
-                            for (boolean close : Arrays.asList(false, true))
-                                arguments.add(Arguments.of(new Args(broken, ignore, blockChecksum, close, payload)));
+                        // Available levels: 1, 2, ... 17.
+                        for (int compressionLevel = 1; compressionLevel < 18; ++compressionLevel)
```

Comment:
Should we use `Lz4OutputStream.MIN_COMPRESSION_LEVEL` and `MAX_COMPRESSION_LEVEL` rather than hardcoding the levels?